--- /usr/share/awesome/lib/awful/client.lua.bak	2012-01-03 12:24:57.335454637 +0100
+++ /usr/share/awesome/lib/awful/client.lua	2012-01-03 12:25:59.635522851 +0100
@@ -477,7 +477,7 @@
         end
         if s > sc then s = 1 elseif s < 1 then s = sc end
         sel.screen = s
-        capi.mouse.coords(capi.screen[s].geometry)
+--        capi.mouse.coords(capi.screen[s].geometry)
     end
 end
 
